### PR TITLE
Add implementation of standard deviation on data

### PR DESCRIPTION
**Pull request by Samuel-Bakes**
Add graph to extend visualisation

### CODE REVIEW

File: inflammation-analysis.py [hunk 1]
```diff
@@ -27,7 +27,8 @@ def main(args):
     for filename in InFiles:
         inflammation_data = models.load_csv(filename)
 
-        view_data = {'average': models.daily_mean(inflammation_data), 'max': models.daily_max(inflammation_data), 'min': models.daily_min(inflammation_data)}
```
Comment (bennetjos) [inline inflammation-analysis.py]: Check the implementation of standard deviation

File: inflammation/models.py [hunk 2]
```diff
@@ -54,3 +54,13 @@ def daily_min(data):
     """Calculate the daily min of a 2d inflammation data array."""
     return np.min(data, axis=0)
 
+
+def s_dev(data):
```
Comment (bennetjos) [inline inflammation/models.py]: Make the function name more explicit about what it is doing

File: inflammation/models.py [hunk 3]
```diff
@@ -54,3 +54,13 @@ def daily_min(data):
...
+    """Computes and returns standard deviation for data."""
+    mmm = np.mean(data, axis=0)
```
Comment (bennetjos) [inline inflammation/models.py]: Avoid using confusing variable name. 
```suggestion
    mean = np.mean(data, axis=0)
```

File: tests/test_models.py [hunk 4]
```diff
@@ -37,3 +37,14 @@ def test_load_from_json(tmpdir):
         temp_json_file.write('[{"observations":[1, 2, 3]},{"observations":[4, 5, 6]}]')
     result = load_json(example_path)
     npt.assert_array_equal(result, [[1, 2, 3], [4, 5, 6]])
+
+
+@pytest.mark.parametrize('data, expected_standard_deviation', [
+    ([0, 0, 0], 0.0),
+    ([1.0, 1.0, 1.0], 0),
+    ([0.0, 2.0], 1.0)
```
Comment (bennetjos) [inline tests/test_models.py:45]: Check the data provided as it requires 3 values, given only 2

File: tests/test_models.py [hunk 5]
```diff
@@ -37,3 +37,14 @@ def test_load_from_json(tmpdir):
...
+])
+def test_daily_standard_deviation(data, expected_standard_deviation):
+    from inflammation.models import s_dev
+    result_data = s_dev(data)['standard deviation']
```
Comment (bennetjos) [inline tests/test_models.py:49]: Check the format of the function call s_dev

**Review by bennetjos — CHANGES_REQUESTED** [review]
Consider all the test cases:
Standard deviation for one patient with multiple observations.
Standard deviation for two patients.
Graph includes a standard deviation graph.
Standard deviation function should raise an error if given empty data.
Computing standard deviation where deviation is different from variance.
Standard deviation function should give correct result given negative inputs.
Function should work with numpy arrays

File: inflammation/models.py (hunk 2 above)
Comment (Samuel-Bakes) [reply inflammation/models.py]: Change has been addressed

**Pull request merged**